### DISCUSSION TITLE
https://github.com/jordanbaird/ice new cask to repository

### DISCUSSION THREAD

**Discussion by lsllit** [Homebrew/discussions#5122] [Casks]
### Output of `brew config`

```shell
ㅤ
```

### Output of `brew doctor`

```shell
ㅤ
```

### Description of issue

oh, sorry i don't know how to open a pull request on https://github.com/Homebrew/homebrew-cask so i am asking for someone to submit a pull request for me.
https://github.com/jordanbaird/ice is a recently published cask used for menu bar management and is useful.

please do a pull request to try adding this cask to the repository.

### Relevant casks

ㅤ

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
